README.md

```diff
@@ -488,7 +488,7 @@ layers:
   - name: Controller
     collectors:
       - type: classNameRegex
-        regex: #.*Controller.*#
```

Comment:
`#` indicates a comment in yaml, therefore the line ending was effectively commented out (and therefore a syntax error)